CHANGELOG.md

```diff
@@ -1,3 +1,285 @@
+# Kubermatic 2.19
+## [v2.19.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.19.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.19/upgrading/guidelines/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure usercluster reconciliations will not cause resource exhausting on seed clusters.
```

Comment:
In place of `exhausting `,  `exhaustion` can be used.

Reply:
done